docs/source/install/docker-compose.rst

```diff
@@ -0,0 +1,275 @@
+=================================
+Docker Container Based Deployment
+=================================
+
+Overview
+********
+
+All the PDS Registry components are containerized as docker images and published in https://hub.docker.com/u/nasapds.
+
+The container based deployment of the PDS Registry components is orchestrated by Docker Compose, a tool for defining and running multi-container Docker applications.
+
+Therefore, the deployment procedures described here can be used to deploy server-side and/or client-side registry components, without having to install the individual components manually.
+
+.. warning::
+ these deployment are not intended for production given that 1. docker compose works on a single host and 2. permanent storage is left to docker to manage.
```

Comment:
Minot suggestion: It is better to start this warning with a upper case T (These) to make it consistent with the .. note: below.

Reply:
Ok, done